apps/docs/components/Navigation/NavigationMenu/NavigationMenu.constants.ts

```diff
@@ -492,6 +492,12 @@ export const PhoneLoginsItems = [
     isDarkMode: true,
     hasLightIcon: true,
   },
+  {
+    name: 'Textlocal SMS Login',
```

Comment:
can we indicate that this is not actively maintained / currently just supported by the community?

Reply:
Yes, updated. Please verify.